assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -105,10 +105,10 @@ function APIProvider({ children }) {
   );
 
   const getMedia = useCallback(
-    ({ mediaType, searchTerm }) => {
+    ({ mediaType, searchTerm, page }) => {
```

Comment:
The `page` is such an overloaded term for us. Can we call it a bit more specifically, e.g. `pagingNum` or such?